Sources/BraintreeCard/CreditCardPOSTBody.swift

```diff
@@ -0,0 +1,136 @@
+// swiftlint:disable all
```

Comment:
We should remove this, we don't want to disable all of our linter rules for any files